src/pages/store/cart.tsx

```diff
@@ -0,0 +1,323 @@
+  }, [cart, credits]);
+
+  const cartInvalid = useMemo(() => cart.some(item => validateClientCartItem(item)), [cart]);
+
+  // prop for item cards to remove their item
+  const removeItem = (optionUUID: UUID) => {
+    setCart(cart => cart.filter(item => item.option.uuid !== optionUUID));
+  };
+
+  // handle confirming an order
+  const placeOrder = () =>
+    placeMerchOrder(cart, pickupEvents[pickupIndex]?.uuid ?? 'x')
+      .then(({ items, pickupEvent }) => {
+        showToast(
+          `Order with ${items.length} item${items.length === 1 ? '' : 's'} placed`,
+          `Pick up at ${pickupEvent.title}`
+        );
+        setCartState(CartState.CONFIRMED);
+        setClientCookie(CookieType.CART, clientCartToCookie([]));
+        setLiveCredits(credits - orderTotal);
+      })
+      .catch(err => {
+        showToast('Unable to place order', getMessagesFromError(err).join('; '));
+      });
+
+  return (
+    <div className={styles.container}>
+      <Navbar balance={liveCredits} showBack />
+      <div className={styles.content}>
+        {/* Title */}
+        <Typography variant="h1/bold" component="h1" className={styles.title}>
+          {cartState !== CartState.CONFIRMED ? 'Your Cart' : 'Order Confirmation'}
+        </Typography>
+
+        {/* Place Order Button */}
+        {cart.length > 0 && cartState !== CartState.CONFIRMED ? (
+          <StoreConfirmModal
+            title="Please confirm you can pick up your order at this event:"
+            opener={
+              <button
+                type="button"
+                className={`${styles.placeOrder} ${
+                  cartState === CartState.CONFIRMING ? styles.confirming : ''
+                }`}
+                onClick={() => {
+                  setCartState(CartState.CONFIRMING);
+                }}
+                disabled={orderTotal > credits || pickupEvents.length === 0 || cartInvalid}
+              >
+                <Typography variant="h4/bold" component="span">
+                  Place Order
+                </Typography>
+              </button>
+            }
+            onConfirm={placeOrder}
+            onCancel={() => setCartState(CartState.PRECHECKOUT)}
+          >
+            {pickupEvents[pickupIndex] && (
+              <EventDetail
+                event={pickupEvents[pickupIndex] as PublicOrderPickupEvent}
+                attended={false}
+                inModal={false}
+              />
+            )}
+          </StoreConfirmModal>
+        ) : (
+          <div />
+        )}
+
+        {/* Confirmation */}
+        {cartState === CartState.CONFIRMED ? (
+          <div className={`${styles.cartCard} ${styles.confirmation}`}>
+            <Typography variant="h3/regular" component="p">
+              You will receive a confirmation email with your order details soon.
+              <br />
+              Please pick up your items on the date you selected. Thank you!
```

Comment:
should it say

```suggestion
              Please pick up your items at the event you selected. Thank you!
```

instead? unsure

Reply:
I like this 👍